lib/electric_slide/call_queue.rb

```diff
@@ -62,7 +62,7 @@ def available_agent_summary
     # @return {Agent}
     def checkout_agent
       agent = @strategy.checkout_agent
-      agent.presence = :busy
+      agent.presence = :busy if agent
```

Comment:
This only guards against updating the agent's presence, but it still returns a nil agent.

We need to figure out how nil agents are getting into the queue in the first place. They should be prevented by this: https://github.com/adhearsion/electric_slide/blob/develop/lib/electric_slide/call_queue.rb#L91